tests/unit/single_table/test_copulas.py

```diff
@@ -0,0 +1,109 @@
+from unittest.mock import patch
+
+import pytest
+from copulas.univariate import BetaUnivariate, GammaUnivariate, UniformUnivariate
+
+from sdv.metadata.single_table import SingleTableMetadata
+from sdv.single_table.copulas import GaussianCopulaSynthesizer
+
+
+class TestGaussianCopulaSynthesizer:
+
+    def test__validate_distribution_str(self):
+        """Test that when a ``str`` is passed, the class from the ``DISTRIBUTIONS`` is returned."""
+        # Setup
+        distribution = 'beta'
+
+        # Run
+        result = GaussianCopulaSynthesizer._validate_distribution(distribution)
+
+        # Assert
+        assert result == BetaUnivariate
+
+    def test__validate_distribution_not_in_distributions(self):
+        """Test that ``ValueError`` is raised when the given distribution is not supported."""
+        # Setup
+        distribution = 'student'
+
+        # Run and Assert
+        with pytest.raises(ValueError, match='Invalid distribution specification student.'):
```

Comment:
I'd put some sort of identifier around the invavlid distribution, otherwise the error message is confusing. Like, `Invalid distribution specification 'student'.` or `Invalid distribution specification (student).`